### DISCUSSION TITLE
Sometimes flying in crosshair mode doesn't work properly.

### DISCUSSION THREAD

**Discussion by G01dY05h1** [Mojang/minecraft-editor#97] [Bugs]
Occasionally, attempting to fly into blocks won't let you fly more than 1 tile into a wall before pushing you back out. I fixed it by going into editor mode and back. 

Unfortunately, I do not know how to reproduce it. All I did was switch from editor mode into crosshair mode.

**Comment by minecraftchloe** [comment]
We'll keep an eye on this during playtests! If you find a reliable way of repro'ing, please reopen the discussion!
